.github/workflows/ci.yml

```diff
@@ -20,3 +20,6 @@ jobs:
       - run: bundle exec rake
         env:
           GOVUK_CONTENT_SCHEMAS_PATH: tmp/govuk-content-schemas
+      - run: bundle exec middleman build --verbose
```

Comment:
Instead of doing this, can we change the `default` rake task to also do `cache:clear` and `assets:precompile`? This avoids duplicating the knowledge.

Reply:
Good idea!